examples/template.html

```diff
@@ -0,0 +1,92 @@
+    <link rel="stylesheet" type="text/css" href="../emperor/support_files/vendor/css/jquery-ui2.css">
+    <link rel="stylesheet" type="text/css" href="../emperor/support_files/vendor/css/slick.grid.min.css">
+    <link rel="stylesheet" type="text/css" href="../emperor/support_files/vendor/css/colorPicker.css">
+    <link rel="stylesheet" type="text/css" href="../emperor/support_files/vendor/css/spectrum.css">
+    <link rel="stylesheet" type="text/css" href="../emperor/support_files/vendor/css/chosen.min.css">
+
+    <style>
+      div.emperor-space{
+        position: fixed;
+        height: 100%;
+        width: 100%;
+        height: 100vh;
+        width: 100vw;
+        margin: 0;
+      }
+    </style>
+
+    <script type="text/javascript" src="../emperor/support_files/vendor/js/underscore-min.js"></script>
+    <script type="text/javascript" src="../emperor/support_files/vendor/js/jquery-1.7.1.min.js"></script>
+    <script type="text/javascript" src="../emperor/support_files/vendor/js/jquery-ui-1.8.17.custom.min.js"></script>
+    <script type="text/javascript" src="../emperor/support_files/vendor/js/jquery.colorPicker.js"></script>
+    <script type="text/javascript" src="../emperor/support_files/vendor/js/chroma.min.js"></script>
+    <script type="text/javascript" src="../emperor/support_files/vendor/js/three.min.js"></script>
+    <script type="text/javascript" src="../emperor/support_files/vendor/js/three.js-plugins/OrbitControls.js"></script>
+
+    <script type="text/javascript" src="../emperor/support_files/vendor/js/jquery.colorPicker.js"></script>
+    <script type="text/javascript" src="../emperor/support_files/vendor/js/spectrum.js"></script>
+
+    <script type="text/javascript" src="../emperor/support_files/vendor/js/chosen.jquery.min.js"></script>
+
+    <!-- slickgrid -->
+    <script type="text/javascript" src="../emperor/support_files/vendor/js/jquery.event.drag-2.2.min.js"></script>
+    <script type="text/javascript" src="../emperor/support_files/vendor/js/slick.core.min.js"></script>
+    <script type="text/javascript" src="../emperor/support_files/vendor/js/slick.grid.min.js"></script>
+    <script type="text/javascript" src="../emperor/support_files/vendor/js/slick.editors.min.js"></script>
+
+    <script type="text/javascript" src="../emperor/support_files/js/model.js"></script>
+    <script type="text/javascript" src="../emperor/support_files/js/view.js"></script>
+    <script type="text/javascript" src="../emperor/support_files/js/util.js"></script>
+    <script type="text/javascript" src="../emperor/support_files/js/sceneplotview3d.js"></script>
+    <script type="text/javascript" src="../emperor/support_files/js/controller.js"></script>
+    <script type="text/javascript" src="../emperor/support_files/js/color-editor.js"></script>
+
+  </head>
+  <body>
+    <script type="text/javascript">
+      // we need to remove this callback from the chosen library
+      function chosen_highlight_callback() {};
```

Comment:
We could, but the thing is that to support a feature in chosen,
@antgonza had to modify the original source and if this callback is not
present, the console is flooded with error messages. So the answer is,
yes but not right now.

On (Aug-19-15|23:18), mortonjt wrote:
> > +    <script type="text/javascript" src="../emperor/support_files/vendor/js/slick.core.min.js"></script>
> > +    <script type="text/javascript" src="../emperor/support_files/vendor/js/slick.grid.min.js"></script>
> > +    <script type="text/javascript" src="../emperor/support_files/vendor/js/slick.editors.min.js"></script>
> > +
> > +    <script type="text/javascript" src="../emperor/support_files/js/model.js"></script>
> > +    <script type="text/javascript" src="../emperor/support_files/js/view.js"></script>
> > +    <script type="text/javascript" src="../emperor/support_files/js/util.js"></script>
> > +    <script type="text/javascript" src="../emperor/support_files/js/sceneplotview3d.js"></script>
> > +    <script type="text/javascript" src="../emperor/support_files/js/controller.js"></script>
> > +    <script type="text/javascript" src="../emperor/support_files/js/color-editor.js"></script>
> > +
> > +  </head>
> > +  <body>
> > +    <script type="text/javascript">
> > +      // we need to remove this callback from the chosen library
> > +      function chosen_highlight_callback() {};
> 
> Can we remove this callback?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/emperor/pull/407/files#r37498139